src/lib.rs

```diff
@@ -146,28 +164,35 @@ static LOGGER: LazyLock<Mutex<Logger>> = LazyLock::new(|| {
     })
 });
 
-/**
-This returns the Log builder with default values.
-This is just a conveniend way to get call [`Builder::default()`].
-[Read more](`crate::Builder`)
-
-Example:
-```rust
-use log::LevelFilter;
-# #[allow(clippy::needless_doctest_main)]
-fn main() {
-    // initialize the logger.
-    // You have to open the ui later within your egui context logic.
-    // You should call this very early in the program.
-    egui_logger::builder()
-        .max_level(LevelFilter::Info) // defaults to Debug
-        .init()
-        .unwrap();
-
-    // ...
+/// Clears all existing retained logs.
+pub fn clear_logs() {
+    LOGGER
+        .lock()
+        .expect("could not get access to logger")
+        .logs
+        .clear();
 }
-```
-*/
+
+/// This returns the Log builder with default values.
+/// This is just a convenient way to get [`Builder::default()`].
+/// [Read more](`crate::Builder`)
+///
+/// Example:
+/// ```rust
+/// use log::LevelFilter;
+/// # #[allow(clippy::needless_doctest_main)]
+/// fn main() {
+///     // Initialize the logger.
+///     // You have to open the ui later within your egui context logic.
+///     // You should call this very early in the program.
+///     egui_logger::builder()
+///         .max_level(LevelFilter::Info) // defaults to Debug
+///         .init()
+///         .unwrap();
+///
+///     // ...
+/// }
+/// ```
```

Comment:
Just wondering, but is there a reason why you switched to per line comments?

Reply:
Ah, that's just because line comments are the more idiomatic/recommended style in Rust (see e.g. [here](https://doc.rust-lang.org/rust-by-example/hello/comment.html) or [here](https://doc.rust-lang.org/style-guide/index.html#comments)), especially for documentation, with block comments being mainly useful to temporarily comment out pieces of code. This isn't a hard rule or anything of course, so if you prefer the block style then I can revert this change.

Reply:
No that's not an issue. I was just curios